graphics/cairo-devel/Portfile

```diff
@@ -68,44 +63,18 @@ if {${configure.build_arch} eq "x86_64" || (${universal_possible} && [variant_is
     compiler.blacklist-append gcc-4.0
 }
 
-configure.args              --disable-gl \
-                            --disable-quartz \
-                            --disable-quartz-font \
-                            --disable-quartz-image \
-                            --disable-silent-rules \
-                            --disable-symbol-lookup \
-                            --disable-xlib \
-                            --disable-xlib-xcb \
-                            --disable-xcb \
-                            --disable-xcb-shm \
-                            --without-x \
-                            --enable-ft \
-                            --enable-pdf \
-                            --enable-png \
-                            --enable-ps \
-                            --enable-script \
-                            --enable-svg \
-                            --enable-tee \
-                            --enable-xml
-
-# Prevent cairo from using ghostscript.
-configure.args-append       ac_cv_prog_GS=""
-
-# Prevent cairo from using lzo2 because its GPL license makes cairo's effective license GPL too.
-configure.args-append       ac_cv_lib_lzo2_lzo2a_decompress=no
-
-variant opengl requires x11 description {Add OpenGL graphics interface} {
-    depends_lib-append      port:mesa
-
-    configure.args-replace  --disable-gl --enable-gl
-    configure.args-append   --enable-glx
-}
```

Comment:
You've removed the opengl variant... Is this functionality no longer available in cairo?

Reply:
I couldn't see it anywhere in the meson_options.txt file. Same is true for all the options I deleted.

Reply:
Ok, the NEWS file says for 1.17.8:

> In a continuing effort to reduce the amount of legacy code, and increase
> the long-term maintainability of cairo, the following backends have been
> removed:
> 
> - GL and GLES drawing